doc/md/community.md

```diff
@@ -0,0 +1,13 @@
+---
+id: slack
+title: Join our Community
```

Comment:
this breaks the old URL please add an alias

Reply:
Sorry. I misread, you can give this a new URL and add the old one as an alias so it doesn't break 😊

Reply:
changed the name to community and added a redirect from slack to community